PortalFetch/crawler_download.py

```diff
@@ -244,8 +278,10 @@ def main():
         fillAdvanceSearch(driver)
         # Save searched courses
         html = saveResult(driver)
-        filename = parser.get('config', 'filename')
-        quarter = parser.get('config', 'quarter')
+        # get Quarter and fileName based On quarter_value in crawler.config
+        filename = generateFilenameAndQuarter(value)[1]
+        quarter = generateFilenameAndQuarter(value)[0]
```

Comment:
You should use `quarter, filename = generateFilenameAndQuarter(value)`

In this case, you won't call the same function for two times. This will reduce the cost of the code.